test/integration/scheduler/scheduler_test.go

```diff
@@ -369,14 +369,19 @@ var _ = ginkgo.Describe("Scheduler", func() {
 
 			ginkgo.By("updating ClusterQueue")
 			updatedCq := &kueue.ClusterQueue{}
-			gomega.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: cq.Name}, updatedCq)).Should(gomega.Succeed())
 
-			updatedCq.Spec.ResourceGroups[0].Flavors[0].Resources[0] = kueue.ResourceQuota{
-				Name:           corev1.ResourceCPU,
-				NominalQuota:   resource.MustParse("6"),
-				BorrowingLimit: pointer.Quantity(resource.MustParse("0")),
-			}
-			gomega.Expect(k8sClient.Update(ctx, updatedCq)).Should(gomega.Succeed())
+			gomega.Eventually(func() error {
+				err := k8sClient.Get(ctx, types.NamespacedName{Name: cq.Name}, updatedCq)
+				if err != nil {
+					return err
+				}
```

Comment:
```suggestion
				gomega.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: cq.Name}, updatedCq)).Should(gomega.Succeed)
```

Should we use the `gomega.Expect()` in the get operation?
IIUC, this operation should succeed every time.

Reply:
It is not in this test scope. It doesn't change test behavior.

Reply:
Previously, line 372 verified whether the get operation succeeds once.

However, with this PR change,  this test passes even if the get operation failed.

So I think it would be better to mark this test as failed if the get operation failed.

Reply:
Here we test the scheduler functionality. Getting operation is relevant for API.

Reply:
It sounds reasonable. I'm ok with this implementation.